tox.ini

```diff
@@ -84,21 +84,17 @@ passenv =
 # These keys are not inherited by any other sections
 #
 setenv =
-    py{37,38,39,310,311}: COVERAGE_FILE = {envtmpdir}/.coverage
-    py{37,38,39,310,311}: BST_TEST_HOME = {envtmpdir}
-    py{37,38,39,310,311}: BST_TEST_XDG_CACHE_HOME = {envtmpdir}/cache
-    py{37,38,39,310,311}: BST_TEST_XDG_CONFIG_HOME = {envtmpdir}/config
-    py{37,38,39,310,311}: BST_TEST_XDG_DATA_HOME = {envtmpdir}/share
-
-    # This is required to run tests with python 3.7
-    py37: SETUPTOOLS_ENABLE_FEATURES = "legacy-editable"
-
+    py{36,37,38,39,310}: COVERAGE_FILE = {envtmpdir}/.coverage
+    py{36,37,38,39,310}: BST_TEST_HOME = {envtmpdir}
+    py{36,37,38,39,310}: BST_TEST_XDG_CACHE_HOME = {envtmpdir}/cache
+    py{36,37,38,39,310}: BST_TEST_XDG_CONFIG_HOME = {envtmpdir}/config
+    py{36,37,38,39,310}: BST_TEST_XDG_DATA_HOME = {envtmpdir}/share
     # This is required to get coverage for Cython
-    py{37,38,39,310,311}-!nocover: BST_CYTHON_TRACE = 1
+    py{36,37,38,39,310}-!nocover: BST_CYTHON_TRACE = 1
```

Comment:
Here and on various other lines, `py311` is accidentally dropped. This needs to be fixed (if we merge this at all).

Reply:
Aha, yes I was confused by other diffs which appeared in the patch but not in `git diff master`, apparently I missed committing some of the changes after resolving conflicts in revert.

Fixed this one along with some other inconsistencies.